### ISSUE TITLE
Error running playbooks with the latest version - only alpha/numeric and underscore (_) characters are supported

### ISSUE THREAD

**Issue by richardjohnallsopp** [dell/dellemc-openmanage-ansible-modules#106]
Hey I'm seeing this issue running any playbook with the latest version:

flynn@NCL-WL-16612:~/ansible-redfish$ ansible-playbook dell_idrac.yml -i inventory/inventory

PLAY [<hostname>] ****************************************************************************************************************************************************************************************

TASK [Gathering Facts] ****************************************************************************************************************************************************************************
ok: [<hostname>]

TASK [idrac_gather_facts : Get <hostname> system inventory] ******************************************************************************************************************
[WARNING]: The value 443 (type int) in a string field was converted to u'443' (type string). If this does not look like what you expect, quote the entire value to ensure it does not change.

fatal: [<hostname>]: FAILED! => changed=false
  msg: 'invalid member name: u''UsbManagedPort '' -- only alpha/numeric and underscore (_) characters are supported'

PLAY RECAP ****************************************************************************************************************************************************************************************
<hostname> : ok=1    changed=0    unreachable=0    failed=1    skipped=0    rescued=0    ignored=0

```
---
# tasks file for idrac_gather_facts

  - name: "Get {{  inventory_hostname  }} system inventory"
    dellemc_get_system_inventory:
     idrac_ip:   "{{ hostname }}"
     idrac_user: "{{ idrac_user }}"
     idrac_pwd:  "{{ idrac_password }}"
    register: facts
    tags:
     - gather_facts
     
  - name: "Output {{  inventory_hostname  }} facts"
    debug: 
     msg: "{{ facts }}"
    tags:
     - gather_facts     
     
  - set_fact:
      controller_fqdn: "{{ (SystemInventory['Controller']|selectattr('FQDD', 'match', '^RAID.+')|first|default({})).get('FQDD', '') }}" 
    tags:
     - gather_facts      
       
       
  - name: "Output {{  inventory_hostname  }} Controller FQDN"
    debug: 
     msg: "{{ controller_fqdn }}"
    tags:
     - gather_facts
```

**Comment by jagadeeshnv** [comment]
Hi @richardjohnallsopp , this issue was due to the update in the enum34 library. this will be seen in enum34==1.1.8 and 1.1.9. We had reported the issue to https://bitbucket.org/stoneleaf/enum34/issues/29/member-keys-with-special-characters. They have fixed it and released 1.1.10. So updating enum34 to 1.1.10 will solve this issue. "pip install enum34==1.1.10"

**Comment by jagadeeshnv** [comment]
@richardjohnallsopp Can we close on this?
